tests/python/unittest/test_tf_model.py

```diff
@@ -1,11 +1,10 @@
 # coding: utf-8
-from dlr.tf_model import TFModelImpl
+from __future__ import print_function
+from dlr import DLRModel
 import numpy as np
+import os
 
-FROZEN_GRAPH_PATH = "/tmp/test_graph.pb"
```

Comment:
Replacing hardcoded `/tmp` with {current_dir} might be useful but I think it should be done in a separate PR.
I'd recommend to make changes in `tf_model.py` and  `test_tf_model.py` only if they are directly related to the goal of this PR, which is introduction of `IDLRModel` interface and `DLRModel` dispatcher

Reply:
I made these changes just for that. We want to present TF and TVM/Treelite test files with uniform API and usage, as users will see them as our example.

Reply:
Ok, I will only change needed API and add a main function.